node/src/main/resources/defaults.conf

```diff
@@ -123,6 +123,27 @@ api-server {
 
   # Enable reporting API
   enable-reporting = false
+
+  # Sets a custom keepalive time, the delay time for sending next keepalive ping
+  keep-alive-time = 2 hours
```

Comment:
Is this means that _keep alive_ pings will be sent every 2 hours? How this relates to _keep-alive-timeout_? Should timeout be set to larger value to detect missing pings?

Reply:
I found the explanation. Timeout is how much server will wait if ping is not received.
> GRPC_ARG_KEEPALIVE_TIMEOUT_MS
This channel argument controls the amount of time (in milliseconds) the sender of the keepalive ping waits for an acknowledgement. If it does not receive an acknowledgment within this time, it will close the connection.

https://github.com/grpc/grpc/blob/master/doc/keepalive.md

Reply:
Maybe we can set keep alive interval to a smaller value, in a matter of minutes?

Reply:
@zsluedem I agree with @tgrospic Lets have keepalive 5 min with timeout 20sec? Its better to notify client that channel failed, e.g. when waiting for propose accomplish or uploading some big deploy.